http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandlerSpec.scala

```diff
@@ -484,6 +484,118 @@ class AzurePubsubHandlerSpec
     res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
+  it should "poll on azure delete vm and disk " in isolatedDbTest {
```

Comment:
This  might not be easy to test (and perhaps outside of the scope of this PR), but wold it be possible to also test the case when the delete `WsmJobStatus` is failed?